test/transferSpec.js

```diff
@@ -120,15 +106,10 @@ describe('Plugin transfers (optimistic)', function () {
       this.pluginA.send(Object.assign({
         id: id,
         amount: '1.0',
-        data: new Buffer(''),
-        noteToSelf: new Buffer(''),
-        executionCondition: undefined,
-        cancellationCondition: undefined,
-        expiresAt: undefined
       }, transferA)).catch(done)
     })
 
-    it('should reject optimistic transfer with repeat id', function (done) {
+    it('should not throw error on optimistic transfer with repeat id', function (done) {
```

Comment:
It would be good if this described what it should actually do. Should it send the payment twice, or should it _not_ send the payment twice but still respond the same?